.gitignore

```diff
@@ -1,3 +1,11 @@
 dist
 sign.key
 .env
+CMakeFiles
+
+CPackConfig.cmake
+CPackSourceConfig.cmake
+Makefile
```

Comment:
I'm assuming these are all generated by the build process, yes?

Reply:
Ya, cmake builds just a couple of files :)